README.md

```diff
@@ -12,24 +12,31 @@ Artifact Registry.
 For more details, see
 https://cloud.google.com/artifact-registry/docs/nodejs/authentication
 
-The module authenticates to Artifact Registry using
-[Google Application Default Credentials](https://developers.google.com/accounts/docs/application-default-credentials).
+The module automatically searches for credentials from the envrironment and authenticates to Artifact Registry. It looks for
+credentials in the following order:
+1. [Google Application Default Credentials](https://developers.google.com/accounts/docs/application-default-credentials).
+2. The current active account logged in via `gcloud auth login`.
+3. If neither of them exist, an error occurs.
 
 NOTE: This module would update credentials for **all** Artifact Registry
 repositories. It would not be suitable if you use multiple account credentials
 in npmrc file.
 
 To use the module:
 
-1.  Log in
+1.  Log in as a service account:
```

Comment:
I think this is confusing since you're mentioning they can also log in as an end user below.

Reply:
Updated with more structure